gulpfile.js

```diff
@@ -632,14 +633,13 @@ function compileUIExtensions(extensionRoot) {
     };
 }
 
-gulp.task("build", ["compileNode"], function() {
-});
+gulp.task("build", gulp.series("compileNode"));
 
 gulp.task("handoff", function(cb) {
     handoff(cb);
 });
```

Comment:
don't we have call cb() like above?

Reply:
i only changed the ones where gulp complained.